bigtable-hbase/pom.xml

```diff
@@ -44,28 +44,6 @@ limitations under the License.
         </dependency>
 
         <dependency>
-            <groupId>com.google.api-client</groupId>
```

Comment:
It's not obvious why this change is here - did closing the snapshot repository with these fail?